client/app/actions/rbac.actions.js

```diff
@@ -0,0 +1,14 @@
+function addRBAC (features) {
+  return {
+    type: 'ADD_RBAC',
+    payload: features
+  }
+}
+function addNavFeatures (features) {
+  return {
+    type: 'ADD_NAVFEATURES',
+    payload: features
+  }
+}
+
+export default { addRBAC, addNavFeatures }
```

Comment:
imho it makes very little sense to export as default multiple functions? it means everywhere you will use it you will have to do map it to multiple variables?